libvast/src/system/default_configuration.cpp

```diff
@@ -31,8 +31,8 @@ default_configuration::default_configuration() {
   set("logger.console-verbosity", defaults::logger::console_verbosity);
   set("logger.file-verbosity", defaults::logger::file_verbosity);
   set("logger.file-format", defaults::logger::file_format);
-  // Allow VAST clusters to form a mesh.
-  set("middleman.enable-automatic-connections", true);
+  // Prevent broker from opening a random public port by default.
+  set("middleman.enable-automatic-connections", false);
```

Comment:
Why don't we just remove these lines instead? `false` is the default after all.

Reply:
But then we'd silently reintroduce the bug if the upstream default changes.

Reply:
This is a question of policy. Either we rely on the defaults or we set everything that affects vast explicitly. So far we've been doing the former.